<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gnutella client</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>by Ferrari Luca, Lodi Giovanni, Bersani Marco</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>611</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Neighbours</string>
     </attribute>
     <widget class="QTreeWidget" name="peersTreeWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>561</width>
        <height>231</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Peer IP</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Peer Port</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Shared files</string>
     </attribute>
     <widget class="QListWidget" name="sharedFilesListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>581</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Downloads</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>File name to search:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchLineEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="searchBtn">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Search!</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="resultsTreeWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>581</width>
        <height>231</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Filename</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Peer IP</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Peer Port</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Double click on a file to download it</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
